extensions/qute/runtime/src/main/java/io/quarkus/qute/runtime/extensions/CollectionTemplateExtensions.java

```diff
@@ -9,7 +9,7 @@
 import io.quarkus.qute.TemplateExtension;
 
 @TemplateExtension
-public class BuiltinTemplateExtensions {
+public class CollectionTemplateExtensions {
```

Comment:
`Map` is not a collection so we should probably extract the relevant methods in `MapTemplateExtensions`.